addons/mail/data/mail_data.xml

```diff
@@ -67,6 +67,52 @@
         <meta itemprop="name" content="View ${ctx['model_name']}"/>
     </div>
 </div>
+
+% if ctx['is_discussion'] and len(ctx['actions']) == 0:
+<div style="padding:0px; width:600px; max-width:600px; margin:0 auto; background: #FFFFFF repeat top /100%;>
```

Comment:
Read like duplicated code. Can't you put each section under an "if" ? Like header if action or not discussion, body in every case, ...